spi/common/web-spi/src/main/java/org/eclipse/edc/web/spi/configuration/WebServiceConfiguration.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright (c) 2020-2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.web.spi.configuration;
+
+/**
+ * WebService configuration returned from {@link WebServiceConfigurator#configure}
+ */
+public class WebServiceConfiguration {
+
+    private final String contextAlias;
+    private final int port;
+    private final String path;
+
+
+    public WebServiceConfiguration(String contextAlias, int port, String path) {
```

Comment:
Since the intention is for this to be created only by the `WebServiceConfigurator`, the constructor should not be `public`.

If it is indeed be user-instantiable, then there should be a `Builder` for it.